eta/core/datasets.py

```diff
@@ -730,6 +730,69 @@ def add_file(self, data_path, labels_path, move_files=False,
 
         return self
 
+    def add_file_and_rename(self, data_path, labels_path, new_data_path,
```

Comment:
I see a lot of duplicate code between this a `add_file`. What about added two new parameters to `add_file` instead?
```
new_data_name=None, new_labels_name=None
```

Reply:
Whoops, meant to do that. Good catch.